conans/client/command.py

```diff
@@ -454,7 +454,8 @@ def install(self, *args):
                                            build=args.build, profile_names=args.profile,
                                            update=args.update, generators=args.generator,
                                            no_imports=args.no_imports,
-                                           install_folder=args.install_folder)
+                                           install_folder=args.install_folder,
+                                           lockfile=args.lockfile)
```

Comment:
Yes, with ``install <ref>``, the arg ``-if`` is only useful if we specify some generators ``-g=cmake`` then it will put the generators there, inside the specified install-folder, otherwise the folder will be empty.

As I don't plan to generate locks with ``conan install`` as the main approach, I think we should leave this as-is to not break users.